src/app/auth/sign-up/sign-up.component.less

```diff
@@ -0,0 +1,57 @@
+@primary-red-color: #F40009;
+
+#signup-section {
+  overflow-x: hidden;
+  margin: 20px auto;
+  width: 500px;
+
+  & p {
+    margin: 20px 0;
+
+    & .sign-in-text {
+      color: @primary-red-color;
+      font-weight: 700;
+
+      &:hover {
+        cursor: pointer;
+        text-decoration: underline;
+      }
+    }
+  }
+}
+
+.progress-row {
+  display: flex;
+  position: relative;
+  left: 5%;
+  margin: 20px auto;
+  width: 40vw;
+
+  & > div {
```

Comment:
Using tags is bad to practise. You need to create an appropriate class for any case